### PR TITLE
Add `fullWidth` prop for `AlphaStack`

### LINKED ISSUE #7208: [Layout foundations] Add full width children prop to Stack component
Add a prop to the `AlphaStack` component that renders all children full width. Currently since it uses `flex` children are flexed and collapse to the size of the content. This can create side effects when nesting components like `Column` within `Stack` that should render full width.

### PR DESCRIPTION
**Pull request by aveline**
### WHY are these changes introduced?

Fixes #7208 

### WHAT is this pull request doing?

Adds a prop to `AlphaStack` that forces children to be full width.

Screen Shot 2022-09-29 at 2 49 11 PM

### CODE REVIEW

**Review by chazdean — APPROVED** [review]
🎩✨

File: polaris-react/src/components/AlphaStack/AlphaStack.tsx [hunk 1]
```diff
@@ -10,18 +10,24 @@ type Align = 'start' | 'end' | 'center';
 export interface AlphaStackProps {
   /** Elements to display inside stack */
   children?: React.ReactNode;
-  /** Adjust spacing between elements */
-  spacing?: SpacingSpaceScale;
   /** Adjust vertical alignment of elements */
   align?: Align;
+  /** Toogle elements to be full width */
```
Comment (laurkim) [inline polaris-react/src/components/AlphaStack/AlphaStack.tsx]:
```suggestion
  /** Toggle elements to be full width */
```

**Review by laurkim — APPROVED** [review]
Nice 🚀

**Pull request merged**